ui/src/config.js

```diff
@@ -71,59 +71,70 @@ export const config = {
           type: 'vocabulary',
           vocabularyType: 'document_accelerators',
           default: '',
-        },
-        'unit:curated_relation': {
-          label: 'Curated Relation',
-          type: 'boolean',
-          default: false,
+          line: 1,
```

Comment:
now I understood the sorting from the previous PR. You've defined the sorting function in `react-invenio-app-ils`, do we have there any extra fields (as a show case)? I'm asking because I didn't see the changed configuration there

Reply:
There are no changes in `react-invenio-app-ils` because the extra fields are [empty](https://github.com/inveniosoftware/react-invenio-app-ils/blob/master/src/lib/config/defaultConfig.js#L224)